src/components/WalletImageSlider.tsx

```diff
@@ -0,0 +1,59 @@
+import { sliderImage } from "./SliderImageWallet";
+import SliderButton from "./SliderButton";
+
+type SliderImageWalletProps = {
+  sliderValue: number;
+  setSliderValue: React.Dispatch<React.SetStateAction<number>>;
+};
+
+export default function WalletImageSlider({
+  sliderValue,
+  setSliderValue,
+}: SliderImageWalletProps) {
+  const SliderValueEqualsIndex = (index: number) => {
+    if (sliderValue >= index) return true;
+  };
+
+  const handleEventChange = (event) => {
+    setSliderValue(event.target.value);
+  };
+
+  return (
+    <div className="flex flex-col justify-center w-1/2 z-40 absolute bottom-0">
+      <div className="relative">
+        <div
+          className={`absolute bg-slider-colour-full h-2 z-40`}
+          style={{
+            width: `${(sliderValue / (sliderImage.length - 1)) * 100}%`,
```

Comment:
Can extract

Reply:
created function that returns % and passed as props to child component.